continuedev/continue
# fix: cmd+shft+l closes sidebar if focused

## Description

Fixes missing functionality and enables cmd+shft+l to close the sidebar if focused. Also removes a now unnecessary `continue.toggleAuxiliaryBar` command.

## Checklist


## Testing
- Don't highlight any code, press `cmd+shft+l`, confirm the side bar opens
- Press `cmd+shft+l`, confirm the sidebar closes
- Highlight code, press `cmd+shft+l`, confirm sidebar opens with code
- Press `cmd+shft+l`, confirm sidebar closes
